### PR TITLE
[AC-1374] Create document structure for payment partners (unzer)

### PR DESCRIPTION
**Pull request by VadymSachenko**
## PR Description

Jira ticket: https://spryker.atlassian.net/browse/AC-1374

### CODE REVIEW

File: docs/pbc/all/payment/unzer/configuration-in-the-back-office/add-unzer-marketplace-credentials.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
```
Comment (andriitserkovnyi) [inline docs/pbc/all/payment/unzer/configuration-in-the-back-office/add-unzer-marketplace-credentials.md:1]: When a section consists only of guides, start the name with a bare infinitive to indicate that. A good name for this section would be `Add Unzer credentials in the Back Office`.

File: docs/pbc/all/payment/unzer/configuration-in-the-back-office/add-unzer-marketplace-credentials.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+| STORE RELATION | Defines stores to add the Unzer payment method to. |
+| MERCHANT REFERENCE | Merchant ID on the Spryker side. |
+| PARTICIPANT ID | Merchant reference on the Unzer side, which identifies a merchant who receives money when the order is paid and gets the status `payment completed`. |
+
+## Add credentials for external merchants
+
+1. Go to **Back Office&nbsp;<span aria-label="and then">></span> Unzer**.
+2. On the **Overview of Unzer Credentials** page, against credentials you want to add external merchant credentials for, click **Edit**.
+  The **Edit Unzer Credentials** page opens.
+3. On the **External Merchant Credentials** tab, click **Add Merchant Unzer Credentials**.
```
Comment (andriitserkovnyi) [inline docs/pbc/all/payment/unzer/configuration-in-the-back-office/add-unzer-marketplace-credentials.md:60]: Do new fields appear or does a page open?

File: docs/pbc/all/payment/unzer/configuration-in-the-back-office/add-unzer-marketplace-credentials.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+4. For the new merchant Unzer credentials, enter a **NAME** <!--the field must be removed from UI-->.
+5. Enter **UNZER PUBLIC KEY**.
+6. Enter **UNZER PRIVATE KEY**.
+7. Select **MERCHANT REFERENCE**.
+8. Enter **PARTICIPANT ID**.
+9. Click **Save**.
```
Comment (andriitserkovnyi) [inline docs/pbc/all/payment/unzer/configuration-in-the-back-office/add-unzer-marketplace-credentials.md:66]: What happens?

File: docs/pbc/all/payment/unzer/configuration-in-the-back-office/add-unzer-marketplace-credentials.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+### Reference information: Add credentials for external merchants
+
+The following table describes the attributes you select and enter when adding Unzer credentials:
```
Comment (andriitserkovnyi) [inline docs/pbc/all/payment/unzer/configuration-in-the-back-office/add-unzer-marketplace-credentials.md:70]: The heading explains this, so you can remove this sentence

Comment (andriitserkovnyi) [reply docs/pbc/all/payment/unzer/configuration-in-the-back-office/add-unzer-marketplace-credentials.md:70]: Apply the comments for the previous table to this one.

File: docs/pbc/all/payment/unzer/configuration-in-the-back-office/add-unzer-standard-credentails.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
+---
```
Comment (andriitserkovnyi) [inline docs/pbc/all/payment/unzer/configuration-in-the-back-office/add-unzer-standard-credentails.md:1]: Apply the comments from the previous do to this one.

File: docs/pbc/all/payment/unzer/domain-model-and-relationships/domain-model-and-relationships.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
+---
+title: Domain model & relationships
```
Comment (andriitserkovnyi) [inline docs/pbc/all/payment/unzer/domain-model-and-relationships/domain-model-and-relationships.md:2]:
```suggestion
title: Domain model and relationships
```

Comment (andriitserkovnyi) [reply docs/pbc/all/payment/unzer/domain-model-and-relationships/domain-model-and-relationships.md:2]: https://developers.google.com/style/word-list?hl=en#letter-symbol

Comment (andriitserkovnyi) [reply docs/pbc/all/payment/unzer/domain-model-and-relationships/domain-model-and-relationships.md:2]: Specify what the model and relationships belong to. There are multiple domain models for different features and customers should be able to find them via search

File: docs/pbc/all/payment/unzer/domain-model-and-relationships/domain-model-and-relationships.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+last_updated: Jun 22, 2022
+description: This document describes domain model & relationships
+template: concept-topic-template
+---
+
+# Domain model for Unzer payments
```
Comment (andriitserkovnyi) [inline docs/pbc/all/payment/unzer/domain-model-and-relationships/domain-model-and-relationships.md:8]: remove

File: docs/pbc/all/payment/unzer/extend-and-customize/frontend-for-a-credit-card.md [hunk 8]
```diff
@@ -0,0 +1,10 @@
+---
```
Comment (andriitserkovnyi) [inline docs/pbc/all/payment/unzer/extend-and-customize/frontend-for-a-credit-card.md:1]: Don't need a separate doc just to give a link. Shall we move it to `unzer.md`?

File: docs/pbc/all/payment/unzer/unzer.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
+---
+title: Unzer
+last_updated: Jun 9, 2022
+description: This document desribes the Unzer payment method
+template: concept-topic-template
+---
+
+## Unzer USP
```
Comment (andriitserkovnyi) [inline docs/pbc/all/payment/unzer/unzer.md:8]: Blank section

File: docs/pbc/all/payment/unzer/install-unzer/integrate-unzer.md [hunk 10]
```diff
@@ -0,0 +1,487 @@
...
+related:
+- title: Install and configure Unzer
+  link: docs/pbc/all/payment/unzer/install-unzer/install-and-configure-unzer.html
+---
+
+This document shows how to integrate the Unzer module into your project.
+
+## Prerequisites
+
+Before integrating Unzer into your project, make sure you have [installed and configured the Unzer module](/docs/pbc/all/payment/unzer/install-unzer/install-and-configure-unzer.html).
```
Comment (andriitserkovnyi) [inline docs/pbc/all/payment/unzer/install-unzer/integrate-unzer.md:15]:
```suggestion
[Install and configure the Unzer](/docs/pbc/all/payment/unzer/install-unzer/install-and-configure-unzer.html).
```

File: docs/pbc/all/payment/unzer/extend-and-customize/frontend-for-a-credit-card.md (hunk 8 above)
Comment (andriitserkovnyi) [reply docs/pbc/all/payment/unzer/extend-and-customize/frontend-for-a-credit-card.md:1]: You can create a next steps sections in that doc and include this link there as well.

**Review by andriitserkovnyi — CHANGES_REQUESTED** [review]

File: docs/pbc/all/payment/unzer/configuration-in-the-back-office/add-unzer-marketplace-credentials.md [hunk 11]
```diff
@@ -0,0 +1,79 @@
...
+
+1. Add Unzer credentials:
+   1. Go to **Back Office&nbsp;<span aria-label="and then">></span> Unzer**.
+   2. On the **Overview of Unzer Credentials** page, click **Add Unzer Credentials**.
+      The **Add Unzer Credentials** page opens.
+   3. For the new credentials, enter a **NAME** .
+   4. From the **CREDENTIALS TYPE** dropdown, select **Marketplace (main channel)**.
+   5. Enter **UNZER PUBLIC KEY**.
+   6. Enter **UNZER PRIVATE KEY**.
+   7. Select **STORE RELATION**.
```
[the remark this answers is not in the document]
Comment (VadymSachenko) [reply docs/pbc/all/payment/unzer/configuration-in-the-back-office/add-unzer-marketplace-credentials.md]: changed to "7. Select **STORE RELATIONS**"

**Review by andriitserkovnyi — APPROVED** [review]

**Pull request merged**